src/styles/sidebar/buttons.scss

```diff
@@ -0,0 +1,22 @@
+// Button styling for the sidebar extending common button-component styles
```

Comment:
I think the file location makes sense here. I'm starting to get mixed up with all the locations of the mixins and button styles. I think it will be easier once we break these into the shared lib.

Reply:
I think that's a fair comment! And a good reminder that I need to pull a bunch of stuff _out_ that currently supports the semi-deprecated `Button` component and other custom buttons.